sdk/resourcemanager/azure-resourcemanager/src/main/java/com/azure/resourcemanager/AzureResourceManager.java

```diff
@@ -952,47 +1234,83 @@ public BlobContainers storageBlobContainers() {
         return this.storageManager.blobContainers();
     }
 
-    /** @return the blob service management API entry point */
+    /**
+     * Gets the blob service management API entry point.
+     *
+     * @return the blob service management API entry point
+     */
     public BlobServices storageBlobServices() {
         return this.storageManager.blobServices();
     }
 
-    /** @return the blob service management API entry point */
+    /**
+     * Gets the blob service management API entry point.
+     *
+     * @return the blob service management API entry point
+     */
     public ManagementPolicies storageManagementPolicies() {
         return this.storageManager.managementPolicies();
     }
 
-    /** @return the spring service management API entry point */
+    /**
+     * Gets the spring service management API entry point.
+     *
+     * @return the spring service management API entry point
+     */
     public SpringServices springServices() {
         return this.appPlatformManager.springServices();
     }
 
-    /** @return the private DNS zone management API entry point */
+    /**
+     * Gets the private DNS zone management API entry point.
+     *
+     * @return the private DNS zone management API entry point
+     */
     public PrivateDnsZones privateDnsZones() {
         return this.privateDnsZoneManager.privateZones();
     }
 
-    /** @return entry point to private endpoints management */
+    /**
+     * Gets entry point to private endpoints management.
+     *
+     * @return entry point to private endpoints management
+     */
     public PrivateEndpoints privateEndpoints() {
         return this.networkManager.privateEndpoints();
     }
 
-    /** @return entry point to tag management management */
+    /**
+     * Gets entry point to tag management management.
+     *
+     * @return entry point to tag management management
+     */
```

Comment:
```suggestion
    /**
     * Gets entry point to tag management.
     *
     * @return entry point to tag management
     */
```

Reply:
Fixed by the [commit#cdcdd09](https://github.com/Azure/azure-sdk-for-java/pull/43638/commits/cdcdd0961d2d9a36d53a311e6dc19451710778e3)